### PR TITLE
Add allCategories endpoint

### PR DESCRIPTION
**Pull request by Justin-Holmes**
https://www.pivotaltracker.com/story/show/154940052

### CODE REVIEW

File: apps/ello_v3/lib/ello_v3/resolvers/categories.ex [hunk 1]
```diff
@@ -0,0 +1,5 @@
+defmodule Ello.V3.Resolvers.Categories do
+  alias Ello.Core.{Discovery}
+
+  def call(_parent, args, _resolver), do: {:ok, Discovery.categories(args)}
```
Comment (alanpeabody) [inline apps/ello_v3/lib/ello_v3/resolvers/categories.ex]: Do we need to worry about pagination here?

Comment (Justin-Holmes) [reply apps/ello_v3/lib/ello_v3/resolvers/categories.ex]: @alanpeabody We don't currently paginate that page, so I think we're fine for now. Looking at it now, we could probably remove several categories from the list.

Comment (alanpeabody) [reply apps/ello_v3/lib/ello_v3/resolvers/categories.ex]: Will the per_page getting passed in from the default args force us to accidentally truncate categories?

Comment (Justin-Holmes) [reply apps/ello_v3/lib/ello_v3/resolvers/categories.ex]: It doesn't look like we are using the `per_page` arg when querying for categories so it should be ignored.

**Pull request merged**